react/features/conference/components/Conference.native.js

```diff
@@ -258,7 +258,9 @@ class Conference extends Component<Props> {
                       * A container that automatically renders indicators such
                       * as VideoQualityLabel or RecordingLabel if need be.
                       */}
-                    <ConferenceIndicators />
+                    {
+                        !this.props._reducedUI && <ConferenceIndicators />
```

Comment:
Please be mindful of consistency! We already have `this.props._reducedUI || <DialogContainer />` a few lines bellow so you should've gone with `this.props._reducedUI || <ConferenceIndicators />`